src/code/z_scene_table.c

```diff
@@ -141,27 +141,27 @@ static EntranceTableEntry* sMayorsResidenceEntranceTable[] = {
 };
 
 static EntranceTableEntry sMajorasLairEntrance0[] = {
-    { 0xF5, 0x00, 0x0387 },
+    { -SCENE_LAST_BS, 0, 0x0387 },
```

Comment:
Do you know what the negative implies?

Reply:
Not 100% sure, the only context is the one function we decompiled in play. I think it has the same purpose as the `ENTRANCE_INFO_CONTINUE_BGM_FLAG` in OoT.